src/food/FoodInfo.ts

```diff
@@ -1,10 +1,19 @@
 import { Material } from 'org.bukkit';
+import { Player } from 'org.bukkit.entity';
 import { ItemStack } from 'org.bukkit.inventory';
+import {
+  eatPoisonousPotato,
+  eatPufferfish,
+  eatRawChicken,
+  eatRottenFlesh,
+  eatSpiderEye,
+  eatSuspiciousStew,
+} from './food-effects';
 
 const BOWL = new ItemStack(Material.BOWL);
 
 // prettier-ignore
-export const FoodInfo = new Map<Material, { foodPoints: number; saturation: number; result?: ItemStack }>([
+export const FoodInfo = new Map<Material, { foodPoints: number; saturation: number; result?: ItemStack; effect?: (player: Player, item?: ItemStack) => void}>([
```

Comment:
```suggestion
export const FOOD_INFO = new Map<Material, { foodPoints: number; saturation: number; result?: ItemStack; effect?: (player: Player, item?: ItemStack) => void}>([
```
Let's fix the naming while we're at it.